js/ui/html_editor/modules/tableResizing.js

```diff
@@ -578,7 +602,11 @@ export default class TableResizingModule extends BaseModule {
                 resultWidth = this._minColumnWidth;
             }
 
-            setLineElementsStyleValue($lineElements, 'width', resultWidth);
+            setLineElementsFormat(this, {
+                lineElements: $lineElements,
```

Comment:
```suggestion
            setLineElementsFormat(this, {
                elements: $lineElements,
```
You already have `line` in the method name